### DISCUSSION TITLE
Gcode thumbnails not working

### DISCUSSION THREAD

**Discussion by Smrts1080** [mriscoc/Ender3V2S1#1267] [General]
Ender 3 neo max all stock hardware 
I updated with the firmware according to the board type 
added tjc update
Changed prusa settings to thumbnail on .png and tried 230x180 and 180x180
Instead of the no image icon ita just blank on files sliced with a thumbnail 
![20240225_123309](https://github.com/mriscoc/Ender3V2S1/assets/161676773/b3515961-7412-457f-8849-800d2ae870eb)

**Comment by Farbaut** [comment]
This is a Dacai display board. Look for the page on how to update the display You want the Dacai display update!

**Reply by Smrts1080** [reply]
Im pretty sure thats the one i did, it wouldnt boot until i did the tjc update it just flickered between 100% and 50% for speed and the rest of the numbers blinked 0 to --

**Comment by Farbaut** [comment]
You are saying you used the TJC update, that is not correct version for the Dacai display board which is in the picture you posted.

**Reply by Smrts1080** [reply]
I downloaded the dacai initially and it wasnt going past the creality screen

**Reply by Smrts1080** [reply]
Ill try reverting to creality firmware then back to DICAI
![Screenshot_20240229_080703_Chrome~2](https://github.com/mriscoc/Ender3V2S1/assets/161676773/c98fe5bf-e4b4-49eb-80a1-c47320cea008)

**Reply by Farbaut** [reply]
Download the Dacai update, unzip and leave only the private folder at the root of the usb card. You could also try a different card that is formatted properly.

**Reply by Smrts1080** [reply]
![20240229_173105 1](https://github.com/mriscoc/Ender3V2S1/assets/161676773/b7045c14-5208-4def-b685-197ada281fb2)
Did what you said got it from the display firmware section of the readme still the same problem. At this point im thinking prusaslicer might be the problem 
![Uploading 20240229_173105.jpg…]()

**Comment by mriscoc** [comment]
Try with the demo G-code file and post the results. Seems to be an slicer's post processing script problem.

**Reply by Smrts1080** [reply]
![17093991251445717397697158425899](https://github.com/mriscoc/Ender3V2S1/assets/161676773/6552cc5e-ea1d-48db-972d-21890a6c7f45)
M808-loops

**Reply by Smrts1080** [reply]
SimpleCuraTest.gcode
![17094037306811039901215380708305](https://github.com/mriscoc/Ender3V2S1/assets/161676773/780e8099-0acd-4eaa-af15-2240938291d2)
SimplePrusaTest
![17094037944244589407996861430892](https://github.com/mriscoc/Ender3V2S1/assets/161676773/fdf4b16f-c137-4881-bffc-ffdaa37cd924)
So yes my prusa slicer isn't working, i did the check box its version 2.7 something

**Comment by Yasamal33** [comment]
PrusaSlicer 2.7.2 x64:

Printer Settings:
![image](https://github.com/mriscoc/Ender3V2S1/assets/15816578/18e3071e-9b5a-47f9-8b5d-71a72f997ccc)

Print settings:
![image](https://github.com/mriscoc/Ender3V2S1/assets/15816578/ccac19bb-310a-42af-baeb-1bde7f42a940)

Windows 10 x64 LTSC

Works fine

**Comment by Koky05** [comment]
Same for me, but with 230x180/PNG.
If you want install python you can use [executable file, created by Giadej](https://github.com/Giadej/PrusaSlicer-Thumbnails-Ender3V2S1) like it was written in Wiki. It works for me too.

**Reply by Smrts1080** [reply]
I have something else going on with prusa slicer i just need to delete everything and start fresh, files sliced witg prusa slicer are messing up and its not happening with creality slicer
